browser/src/control/Control.PartsPreview.js

```diff
@@ -60,6 +60,7 @@ L.Control.PartsPreview = L.Control.extend({
 		map.on('deletepage', this._deletePreview, this);
 		map.on('scrolllimits', this._invalidateParts, this);
 		map.on('scrolltopart', this._scrollToPart, this);
+		map.on('beforequestpreview', this._beforeRequestPreview, this);
```

Comment:
can we take the opportunity to use 'beforerequestpreview' instead of 'beforequestpreview' to avoid confusion later

Reply:
Ups, typo, thanks